RIck and Morty/Model/NetworkManager.swift

```diff
@@ -31,8 +31,10 @@ class NetworkManager {
         do {
           let results = try decoder.decode(Results.self, from: data)
           self.hasNextPage = results.info.next != ""
-          
-          completionHandler(results.results)
+
+          DispatchQueue.main.async {
```

Comment:
this manager switches to a background thread, it might as well switch back to the main thread